### PR TITLE
Prepare release version

### PR DESCRIPTION
**Pull request by milafrerichs**
## What does this PR do?
Prepares for release as a standalone tool. 

Remove the debugger from the viewer and use the default geonode viewer.
Build the viewer.jsx which does hold the abstraction from the react components.
Next step is to release all dependencies as a package and upload to npm to be used with geonode.

### CODE REVIEW

File: viewer.jsx [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
   constructor(domId, config) {
     this.domId = domId;
     this.mapConfig = config;
+    this.proxy = undefined;
   }
   set config(value) {
-    this.mapConfig = config;
+    this.mapConfig = value;
+  }
+  set setProxy(value) {
```
Comment (bartvde) [inline viewer.jsx]: should both setters have the same signature? i.e. if this is called setProxy, should the other one be called setMapConfig instead?

Comment (milafrerichs) [reply viewer.jsx]: yes, you're right. I addressed that a little different in the next commit.

**Comment by milafrerichs** [comment]
what do you think about the new setters @bartvde

**Comment by bartvde** [comment]
perfect thanks @milafrerichs

**Review by bartvde — APPROVED** [review]

**Comment by milafrerichs** [comment]
thanks for your review @bartvde

**Pull request merged**